portfolio/src/main/webapp/index.html

```diff
@@ -1,13 +1,27 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
Nice addition. Minor nit: could be en-US

Reply:
fixed it